melodic-manatees/early_internet/background_app/views.py

```diff
@@ -0,0 +1,107 @@
+from .forms import FileUploadForm
+from .models import BackgroundFile
+from users.models import UserProfile
+from . import background_utility
+import os
+from django.core.files import File
+from django.shortcuts import render, redirect
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
```

Comment:
It helps to make imports more readable if you group them into three groups - standard library imports first, then third party imports, and then local imports.

```py
import os

from django.core.files import File
from django.shortcuts import render, redirect
from django.contrib import messages
from django.contrib.auth.decorators import login_required

from users.models import UserProfile
from .forms import FileUploadForm
from .models import BackgroundFile
from . import background_utility